src/ess/loki/general.py

```diff
@@ -205,16 +88,32 @@ def detector_lab_frame_transform(
     return LabFrameTransform[ScatteringRunType](detector[transform_path])
 
 
-providers = (
+loki_providers = (
     detector_pixel_shape,
     detector_lab_frame_transform,
-    calibrate_detector,
-    get_detector_data,
-    get_monitor_data,
-    get_sample_position,
-    get_source_position,
-    assemble_detector_data,
-    assemble_monitor_data,
     data_to_tof,
     monitor_to_tof,
 )
+
+
+def LokiAtLarmorWorkflow() -> sciline.Pipeline:
```

Comment:
Does it make more sense to move this to the `workflow.py` file?
If so, this file would now be pretty empty. Can we remove it altogether? (move stuff to workflow.py or __init__.py?)